commons/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="fr.free.nrw.commons"
+    package="org.wikimedia.commons"
```

Comment:
We unfortunately can not use this package name, as it has been used already on Google Play and is thus unusable. I tried and it was rejected, that's why I used this unofficial-looking package name.